bin/dsmd.pl

```diff
@@ -547,14 +547,19 @@ sub run {
     );
     $self->{db_centstorage}->connect();
 
-    $self->load_slot_locks();
+    my $moduleDsm = $self->{db_centstorage}->query("select count(name) from centreon.modules_informations where name = 'centreon-dsm'");
+    if ($moduleDsm->fetchrow() != 0) {
+        $self->load_slot_locks();
+    }
     while (1) {
         $self->check_signals();
         $self->clean_locks();
-        if ($self->get_alarms() == 0) {
-            $self->manage_alarms();
+        my $moduleDsmCacheExist = $self->{db_centstorage}->query("select count(*) FROM information_schema.TABLES WHERE TABLE_SCHEMA = 'centreon_storage' AND TABLE_NAME = 'mod_dsm_cache'");
```

Comment:
You can't hardcoded `centreon_storage` database name
Some users will use databases with other names

Reply:
furthermore, you are doing this query in a `while (1)`, this is a performance regression ...

Reply:
I'll delete this modification, the table 'mod_dsm_cache' is created after installing module from website